src/menu-items/attachments-link.tsx

```diff
@@ -0,0 +1,16 @@
+import { ConfigurableLink } from "@openmrs/esm-react-utils";
+import React from "react";
+import { useTranslation } from "react-i18next";
+
+export default function({ basePath }) {
+  const { t } = useTranslation();
+
+  return (
+    <ConfigurableLink
+      to={`${basePath}/attachments`}
```

Comment:
@samuelmale it looks like the basePath itself must include the last "/" as the path contains "//". 
<img width="1233" alt="image" src="https://user-images.githubusercontent.com/139934/104255535-29b6b280-542e-11eb-89a1-f5afe3f4b8eb.png"> 

This is true for all the links you've created.

Reply:
Uh-oh! ConfigurableLink is supposed to already deal with that intelligently. @samuelmale is using this correctly. I'll look into it.

Reply:
@jonathandick this is happening because the `basePath` has an extra forward slash which I fixed [here](https://github.com/openmrs/openmrs-esm-patient-chart/pull/204/files#diff-fdcb51c6cb31fcfecd78b958740498466c84e2f38ff9e47a1056e74c44f4a582R12-R14) but the associated PR https://github.com/openmrs/openmrs-esm-patient-chart/pull/204 isn't merged in yet. Seems @FlorianRappl just forgot about it.